addon-resizer/nanny/main/pod_nanny.go

```diff
@@ -163,6 +163,7 @@ func loadNannyConfiguration(configDir string, defaultConfig *nannyconfigalpha.Na
 	} else if configMapConfig, err = decodeNannyConfiguration(data, scheme, codecs); err != nil {
 		glog.V(0).Infof("Unable to decode Nanny Configuration from config map, using default parameters")
```

Comment:
I think we need to set:
configMapConfig = &nannyconfigalpha.NannyConfiguration{}
Otherwise configMapConfig will be nil and line 166 will fail.